src/shipping/views/WeightRatesUpdate/WeightRatesUpdate.tsx

```diff
@@ -81,53 +82,72 @@ export const WeightRatesUpdate: React.FC<WeightRatesUpdateProps> = ({
     variables: { id, ...paginationState }
   });
 
+  const rate = data?.shippingZone?.shippingMethods.find(
+    rate => rate.id === rateId
```

Comment:
We have a function for that, `getById` 
```suggestion
    data?.shippingZone?.shippingMethods.find(getById(rateId))
```